src/css/grid.css

```diff
@@ -74,177 +74,102 @@ ul {
   display: block;
   text-align: center;
 }
-@media (min-width: 768px) {
-  .container {
-    display: grid;
-    min-height: 100vh;
-    min-width: 100vw;
-    /* background-color: blue; */
-    /* gap: 5px; */
-    grid-template-areas:
-      "top    top     top"
-      "nav    nav     nav"
-      "midl   midm    midr"
-      "bot    bot     bot";
-}
-
-.header-row {
-    grid-area: top;
-    background-color: #f0f0f0;
-    padding: 10px;
-}
-
-.logo {
-    margin-left: auto;
-    margin-right: auto;
-    display: block;
-}
-
-.primary-header {
-    font-size: 22px;
-    text-align: center;
-    padding-bottom: 3px;
-}
-
-.navbar {
-    background-color: #c0c0c0;
-    grid-area: nav;
-}
-
-ul {
-    list-style-type: none;
-    padding: 8px;
-    text-align: center
-}
-
-.link {
-    text-decoration: none;
-    color: black;
-}
-
-.li {
-    display: inline;
-    padding: 8px;
-
-}
-
-.main {
-    grid-area: mid;
-    background-color: white;
-    margin: 1%;
-}
-
-.heading {
-    font-weight: bold;
-    font-size: 18px;
-    padding: 5px;
-}
 
-.content {
-    margin-top: 10px;
-    padding: 5px;
-}
-
-.footer {
-    grid-area: bot;
-    background-color: #c0c0c0;
-    display: block;
-    text-align: center;
-}
+@media (min-width: 768px) {
+    .container {
+        display: grid;
```

Comment:
duplicate declaration - `display` has already been declared in the previous `.container` class.  You only need to add new or changed properties.

Reply:
Ohhhh. This makes sense, I will fix this.